domains/ajays.json

```diff
@@ -1,11 +1,11 @@
 {
   "description": "want to use it for hosting my portfolio site",
-  "repo": "https://github.com/AjayIsDev/AjayIsDev.github.io",
+  "repo": "https://github.com/Dev-Ajay-sahani/Ajay_site",
   "owner": {
-    "username": "AjayIsDev",
+    "username": "Dev-Ajay-sahani",
     "email": "ajays.dev.contact@gmail.com"
   },
   "record": {
-    "CNAME": "AjayIsDev.github.io"
+    "CNAME": "Dev-Ajay-sahani.github.io"
```

Comment:
Visited the repo and it doesn't seem like you have only "hi", please add more stuff to your website